### PR TITLE
Fix enchantment table overlay

### PR DESCRIPTION
**Pull request by NopoTheGamer**

### CODE REVIEW

**Review by DeDiamondPro — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/hex/ItemType.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	ItemType() {}
+
+	ItemType(int starLevel) {
+		this.starLevel = starLevel;
+	}
+
+	public int getStarLevel() {
+		return starLevel;
+	}
+}
```
Comment (DeDiamondPro) [inline src/main/java/io/github/moulberry/notenoughupdates/miscgui/hex/ItemType.java:80]: Shouldn't this be part of repo? might need to be updated later with new updates

**Review by DeDiamondPro — CHANGES_REQUESTED** [review]
L nopo

**Review by DeDiamondPro — DISMISSED** [review]

**Review by Lulonaut — APPROVED** [review]
seems to work (I do not have the hex thing)

**Comment by NopoTheGamer** [comment]
why must that dismiss reviews :sob:

**Review by Lulonaut — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Comment by hannibal00212** [comment]
Don't gonna review or approve a 4700 lines class

**Comment by NopoTheGamer** [comment]
> Don't gonna review or approve a 4700 lines class

:+1: I'll just get jani to look at the code then Kappa

**Comment by jani270** [comment]
> > Don't gonna review or approve a 4700 lines class
> 
> 👍 I'll just get jani to look at the code then Kappa

Ok i will

**Review by hannibal00212 — APPROVED** [review]
I dont like big classes

**Pull request merged**